src/task-definitions.js

```diff
@@ -0,0 +1,347 @@
+import NotebookTask from './notebook-task'
+import { store } from './store'
+import actions from './actions'
+import { isCommandMode,
+  viewModeIsEditor,
+  getCellBelowSelectedId,
+  getCellAboveSelectedId } from './notebook-utils'
+
+const dispatcher = {}
+for (const action in actions) {
+  if (Object.prototype.hasOwnProperty.call(actions, action)) {
+    dispatcher[action] = (...params) => (store.dispatch(actions[action](...params)))
+  }
+}
+
+const oscpu = window.navigator.oscpu || window.navigator.platform
+let OSName = 'Unknown OS'
+if (oscpu.indexOf('Win') !== -1) OSName = 'Windows'
+if (oscpu.indexOf('Mac') !== -1) OSName = 'MacOS'
+if (oscpu.indexOf('X11') !== -1) OSName = 'UNIX'
+if (oscpu.indexOf('Linux') !== -1) OSName = 'Linux'
+
+function commandKey(key) {
+  let ctr = 'Ctrl '
+  if (OSName === 'MacOS') {
+    ctr = '⌘ '
+  }
+  return ctr + key
+}
+
+const TASKS = {}
+
+TASKS.EvaluateCell = new NotebookTask({
+  title: 'Evaluate Cell',
+  keybindings: ['mod+enter'],
+
+  keybindingCallback() {
+    dispatcher.changeMode('command')
+    dispatcher.evaluateCell()
+  },
+})
+
+TASKS.EvaluateAndSelectBelow = new NotebookTask({
+  title: 'Evaluate and Select Below',
+  keybindings: ['shift+enter'],
+
+  keybindingCallback() {
+    if (viewModeIsEditor()) {
```

Comment:
I completely agree. I really don't like that we're using both `callback` and `keybindingCallback`. These should be unified, since for the most part these are duplicates of each other w/ a couple of additional checks for `keybindingCallback`. This would be MUCH cleaner without two callbacks.

Reply:
Will have to think a bit more about how this should be best-approached.